src/request_handler.rs

```diff
@@ -59,6 +69,16 @@ impl RequestHandler {
 
     Ok(Response::new(Body::from(body.into_string())))
   }
+
+  async fn serve_file(&self, file_path: &str) -> Result<Response<Body>> {
+    // todo: stream files
+    let file_contents = tokio::fs::read(dbg!(self
```

Comment:
dbg statement

(Actually it probably isn't necessary to mention stuff that will be caught by the forbid script)